src/test/scala/com/microsoft/hyperspace/index/rankers/RankerTestHelper.scala

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (2020) The Hyperspace Project Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.microsoft.hyperspace.index.rankers
+
+import org.apache.spark.sql.catalyst.expressions.{Attribute, AttributeReference}
+import org.apache.spark.sql.types.{StructField, StructType}
+
+import com.microsoft.hyperspace.actions.Constants
+import com.microsoft.hyperspace.index.{Content, CoveringIndex, Directory, FileInfo, Hdfs, IndexLogEntry, LogicalPlanFingerprint, Relation, Signature, Source, SparkPlan}
```

Comment:
nit: Does the styling guide say anything about these kinds of long import lines? :)

Reply:
import statement is the exception. :)

Reply:
Got it. I was hoping there's some additional guidelines but fine 🙂